pkg/k8s/clientFactory.go

```diff
@@ -58,26 +58,26 @@ type clientFactory struct {
 func NewClientFactory(config *rest.Config, resyncPeriod time.Duration) ClientFactory {
 	stewardClientset, err := steward.NewForConfig(config)
 	if err != nil {
-		log.Printf("could not create Steward clientset: %s", err)
+		klog.V(2).Infof("could not create Steward clientset: %s", err)
 		return nil
 	}
 	stewardInformerFactory := stewardinformer.NewSharedInformerFactory(stewardClientset, resyncPeriod)
 
 	kubernetesClientset, err := kubernetes.NewForConfig(config)
 	if err != nil {
-		log.Printf("could not create Kubernetes clientset: %s", err)
+		klog.V(2).Infof("could not create Kubernetes clientset: %s", err)
```

Comment:
IMHO, it would be more intuitive to add _named constant_ instead of plane numbers.
#### Suggestion: 
```
const (
     VERBOSITY_INFO_QUIET = iota - 1
     VERBOSITY_INFO_DEFAULT
     VERBOSITY_INFO_NORMAL
     VERBOSITY_INFO_VERBOSE
     VERBOSITY_INFO_EXTENDED
     VERBOSITY_INFO_DEBUG
     VERBOSITY_INFO_TRACE = 5
)

klog.V(VERBOSITY_INFO_VERBOSE).Infof("could not ....")
```

Reply:
If we do so, I would vote for shorter names. Yes, variables and constants should express what's their purpose, but in this case in this case readability is more important from my perspective.

Reply:
We would need to define them in an own package.

The proposed all-uppercase names are not in line with Go conventions.

Reply:
Maybe we should stick to the conventions out there, which is the plain numbers. 🤔

Reply:
> Maybe we should stick to the conventions out there, which is the plain numbers. thinking

I got the same feeling.

Reply:
From [go-logr/logr](https://github.com/go-logr/logr#differences-from-daves-ideas):
> Verbosity-levels on info logs. This gives developers a chance to indicate arbitrary grades of importance for info logs, without assigning names with semantic meaning such as "warning", "trace", and "debug". Superficially this may feel very similar, but the primary difference is the lack of semantics. Because verbosity is a numerical value, it's safe to assume that an app running with higher verbosity means more (and less important) logs will be generated.

Reply:
And further:
> What if my V-levels differ between libraries?
>
> That's fine. Control your V-levels on a per-logger basis, and use the WithName function to pass different loggers to different libraries.
>
> Generally, you should take care to ensure that you have relatively consistent V-levels within a given logger, however, as this makes deciding on what verbosity of logs to request easier.

And:

> How do I choose my V-levels?
>
> This is basically the only hard constraint: increase V-levels to denote more verbose or more debug-y logs.
>
> Otherwise, you can start out with 0 as "you always want to see this", 1 as "common logging that you might possibly want to turn off", and 10 as "I would like to performance-test your log collection stack".
>
> Then gradually choose levels in between as you need them, working your way down from 10 (for debug and trace style logs) and up from 1 (for chattier info-type logs).